src/library/scala/collection/Iterable.scala

```diff
@@ -850,19 +850,22 @@ object IterableOps {
     * These operations are implemented in terms of
     * [[scala.collection.IterableOps.sizeCompare(Int) `sizeCompare(Int)`]].
     */
-  final class SizeCompareOps private[collection](val it: IterableOps[_, AnyConstr, _]) extends AnyVal {
+  final class SizeCompareOps private[collection](private val coll: IterableOps[_, AnyConstr, _]) extends AnyVal {
```

Comment:
just `(@deprecated("you can't see me blushing","") val it`. FSR `it` changes `its` name from `it` to `it$extension`.

Reply:
well. that's frustrating. unfortunately, doing what you suggest means there are six unavoidable deprecation warnings for its use within the value class